reference_designs/core/add_one_objFifo_no_mem_tile/run_vck5000.lit

```diff
@@ -0,0 +1,10 @@
+// (c) Copyright 2023 Advanced Micro Devices, Inc.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// REQUIRES: hsa
+//
+// RUN: %python %S/aie2.py xcvc1902 6 > ./aie.mlir
+// RUN: %python aiecc.py %VitisSysrootFlag% --host-target=%aieHostTargetTriplet% %link_against_hsa% ./aie.mlir -I%host_runtime_lib%/test_lib/include %extraAieCcFlags% %S/test_vck5000.cpp -o test.elf -L%host_runtime_lib%/test_lib/lib -ltest_lib
+// RUN: %run_on_vck5000 ./test.elf | FileCheck %s
```

Comment:
We can't use `FileCheck` here, because `%run_on_vck5000` will be set to `echo` when `hsa` is found but vck5000 is not found (i.e. build only). Instead we rely on the exit code of `%run_on_vck5000 test.elf`

```
RUN: at line 9: echo ./test.elf | /scratch/jefff/acdc/build/llvm/bin/FileCheck /scratch/jefff/acdc/aie/reference_designs/core/vector_scalar_kernel/run_vck5000.lit
+ echo ./test.elf
+ /scratch/jefff/acdc/build/llvm/bin/FileCheck /scratch/jefff/acdc/aie/reference_designs/core/vector_scalar_kernel/run_vck5000.lit
/scratch/jefff/acdc/aie/reference_designs/core/vector_scalar_kernel/run_vck5000.lit:10:11: error: CHECK: expected string not found in input
// CHECK: PASS!
          ^
<stdin>:1:1: note: scanning from here
./test.elf
^

Input file: <stdin>
Check file: /scratch/jefff/acdc/aie/reference_designs/core/vector_scalar_kernel/run_vck5000.lit

-dump-input=help explains the following input dump.

Input was:
<<<<<<
          1: ./test.elf 
check:10     X~~~~~~~~~~ error: no match found
>>>>>>

--

********************
********************
Failed Tests (10):
  AIE_REFERENCE_DESIGNS :: core/add_one_objFifo_no_mem_tile/run_vck5000.lit
  AIE_REFERENCE_DESIGNS :: core/matrix_add_one/run_vck5000.lit
  AIE_REFERENCE_DESIGNS :: core/passthrough_hardware/run_vck5000.lit
  AIE_REFERENCE_DESIGNS :: core/vector_add/run_vck5000.lit
  AIE_REFERENCE_DESIGNS :: core/vector_max/run_vck5000.lit
  AIE_REFERENCE_DESIGNS :: core/vector_min/run_vck5000.lit
  AIE_REFERENCE_DESIGNS :: core/vector_mult/run_vck5000.lit
  AIE_REFERENCE_DESIGNS :: core/vector_scalar/run_vck5000.lit
  AIE_REFERENCE_DESIGNS :: core/vector_scalar_kernel/run_vck5000.lit
  AIE_REFERENCE_DESIGNS :: core/vector_sum/run_vck5000.lit
```

Reply:
> We can't use `FileCheck` here, because `%run_on_vck5000` will be set to `echo` when `hsa` is found but vck5000 is not found (i.e. build only). Instead we need to rely on the exit code of `%run_on_vck5000 test.elf`

We should be able to have a build-only github actions workflow here, with rocm installed, [like in mlir-air](https://github.com/Xilinx/mlir-air/blob/main/.github/workflows/buildAndTestAieTools.yml).

Reply:
Great catch thanks, will update this and make sure the build-only test works before merging.